gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/DomainServiceImpl.java

```diff
@@ -359,6 +360,15 @@ public Single<Domain> create(String organizationId, String environmentId, NewDom
                                 .andThen(Single.defer(() -> domainRepository.create(domain)));
                     }
                 })
+
+                .onErrorResumeNext(ex -> {
+                    if (ex instanceof AbstractManagementException) {
+                        return Single.error(ex);
+                    }
+
+                    LOGGER.error("An error occurred while trying to create a domain", ex);
+                    return Single.error(new TechnicalManagementException("An error occurred while trying to create a domain", ex));
+                })
```

Comment:
**question**: what is the impact of moving this from the end of the reactive flow to here if the next steps are failing ?

Reply:
looks like someting got lost in the rebase - resumeNext should stay at the end, there's no reason to move it